api/lib/infrastructure/repositories/certification-course-repository.js

```diff
@@ -90,6 +90,13 @@ module.exports = {
 
     return !exist;
   },
+
+  async findCertificationCoursesBySessionId({ sessionId }) {
+    return CertificationCourseBookshelf
+      .where({ sessionId })
+      .fetchAll()
+      .then((results) => bookshelfToDomainConverter.buildDomainObjects(CertificationCourseBookshelf, results));
+  },
```

Comment:
La fonction est `async` alors qu'il n'y pas d'`await`. Je pense d'ailleurs qu'on privilégie les `async / await` que le Promise chaining
```suggestion
  async findCertificationCoursesBySessionId({ sessionId }) {
    const bookshelfCertificationCourses = await CertificationCourseBookshelf
      .where({ sessionId })
      .fetchAll();
    return bookshelfToDomainConverter.buildDomainObjects(CertificationCourseBookshelf, bookshelfCertificationCourses);
  },
```